components/FAQ/FAQPage/index.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useTheme } from 'emotion-theming'
+import { useContext } from 'react'
+import { Container} from 'reactstrap'
+import Title from 'components/Utils/Title'
+import { DarkContext } from 'helpers/user'
+import { styles } from './styles'
+import QuestionCard from '../QuestionCard'
+
+const questionList = [
+    {
+        id:'external_member_signup',
+        description:'Who can sign up to be CFC member?',
+        answer:'Our membership is not limited to UWA student. Everybody who has a heart for software dev is welcome to be part of us.'
+    },
+    {
+        id:'membership_fee',
+        description:'How much do I pay to be a member?',
+        answer:'$5 :) either via <a target="_blank" href="https://cfcmemberships.getqpay.com/">Qpay<\/a> or contact our friendly committee.'
+    },
+    {
+        id:'project_duration',
+        description:'Do you guys run projects during semester as well?',
+        answer:'Our project is open source, so feel free to contribute whenever. If you need help to start the project, please reach out to committee we are happy to help.'
+    },
+    {
+        id:'during_semester',
+        description:'What do you offer during semester then?',
+        answer:'We run workshop and event during semester.'
+    },
+    {
+        id:'stack_choice',
+        description:'What stack do you guys intend on using for project?',
+        answer:'We work with Javascript, Typescript and Node. Framework will most likely be Vue and Nuxt.js with express, or React with Next.js'
+    },
+    {
+        id:'inexperienced_newbie',
+        description:'Is it recommended to apply if I don\'t have any experience with html/css/javascript?',
+        answer: 'It\'s strongly recommended but not necessary. The reason for that is we work with technologies that build on top of those basic language. Without knowing the basic, it might be hard to grasp the concept. If you would like some guidance for learning, feel free to head to #coding-resources section in our Discord server! '
```

Comment:
```suggestion
        answer: 'It\'s strongly recommended but not necessary. The reason for that is we work with technologies that build on top of those basic fundamentals. Without knowing the basics, it might be hard to grasp more difficult concepts. If you would like some guidance on where to start, head to the #coding-resources section in our Discord server! '
```

Add a link to the Discord server as well I reckon.